Parse/src/main/java/com/parse/Parse.java

```diff
@@ -369,7 +370,7 @@ public static void initialize(Context context, String applicationId, String clie
     );
   }
 
-  public static void initialize(Configuration configuration) {
+  public static void initialize(final Configuration configuration) {
```

Comment:
let's just hold a `final` reference of `Context` instead of the whole `configuration`